update-git-submodules/src/index.test.ts

```
@@ -174,6 +174,60 @@ describe("configureAuth / cleanupAuth", () => {
+			okExec(
+				[
+					" abc1234567890abcdef1234567890abcdef12345 vendor/a (heads/main)",
+					"-bcd1234567890abcdef1234567890abcdef12345 vendor/b",
+					"-cde1234567890abcdef1234567890abcdef12345 vendor/c",
+				].join("\n"),
+			),
+		);
+		const { assertSubmodulesInitialized } = await import("./index.js");
+		await expect(
+			assertSubmodulesInitialized(["vendor/a", "vendor/b", "vendor/c"]),
+		).rejects.toThrow(/vendor\/b, vendor\/c/);
```

ESLint

[error] Move this regular expression to module scope to avoid re-compilation on every call.

(e18e/prefer-static-regex)